src/cmd-buildextend-installer

```diff
@@ -206,17 +210,47 @@ def generate_iso():
         genisoargs += ['-r', '-l', '-sysid', 'PPC',
                        '-chrp-boot', '-graft-points']
     elif basearch == "s390x":
+        INITRD_ADDRESS = '0x02000000'
+        lorax_templates = '/usr/share/lorax/templates.d/99-generic/config_files/s390'
```

Comment:
I think we need to add `lorax-templates-generic` to `deps-s390x.txt` for this, no?

Reply:
`lorax` rpm will pull in that package by default

Reply:
Ahh heh, the first thing I did before my comment was `rpm -qa | grep lorax` and assumed it wasn't a dep already. But of course, it is a dep but for s390x only, heh.